verifier/Cargo.toml

```diff
@@ -22,5 +22,6 @@ std = ["air/std", "vm-core/std", "winter-verifier/std"]
 
 [dependencies]
 air = { package = "miden-air", path = "../air", version = "0.8", default-features = false }
+tracing = "0.1.40"
```

Comment:
Should we change this to `tracing = { version = "0.1", default-features = false, optional = true }` as we do in other files?